src/libs/ActiveClientManager/index.js

```diff
@@ -24,7 +24,9 @@ function isReady() {
 Onyx.connect({
     key: ONYXKEYS.ACTIVE_CLIENTS,
     callback: (val) => {
-        activeClients = !val ? [] : val;
+        if (val) {
+            activeClients = _.unique(activeClients.concat(val));
```

Comment:
Suggestion: Use `if (val && activeClients.indexOf(value) < 0)` instead of `_.unique()`. I think it would be better to do if you're trying to prevent the same value being put into the array again.

(sorry, I'm jaded against `_.unique()` after learning that it's not very performant).

Reply:
Not that it matters much here, since at most there will be 20 items, but updated